partiql-plan/build.gradle.kts

```diff
@@ -25,6 +26,17 @@ dependencies {
     implementation(Deps.kotlinReflect)
 }
 
+// Disabled for partiql-plan project.
```

Comment:
New in this PR: disable expclit api mode for the generated code

Reply:
AFAICT v0.10.0 (and v0.10.1 since v0.10.1 is based off v0.10.0) didn't have the explicit API change yet. Explicit API mode seems to have been enabled for the v0.11.0 release: https://github.com/partiql/partiql-lang-kotlin/pull/1081